test/sanity/validate-modules/skip.txt

```diff
@@ -1,2 +0,0 @@
-lib/ansible/modules/utilities/logic/async_status.py
```

Comment:
This file shouldn't be modified.

Reply:
Hi @mattclay 
Yeap, I've guessed about it, but I can't understand what this test expecting from me:
test/sanity/validate-modules/ignore.txt:538:1: A102 Remove since "lib/ansible/modules/network/ios/ios_logging.py" passes "E322" test

Reply:
It's asking you to remove `lib/ansible/modules/network/ios/ios_logging.py` from line 538 of `test/sanity/validate-modules/ignore.txt` because the module now passes rule E322.

Reply:
Oh, I'm understand, but my file contains only two strings.
lib/ansible/modules/utilities/logic/async_status.py
lib/ansible/modules/utilities/helper/_accelerate.py
Maybe I should pull changes from mail ansible repo to my fork?

Reply:
Hmm, great, now all test passed. )